src/component-library/types/index.tsx

```diff
@@ -1,3 +1,5 @@
-type Tokens = 'BTC' | 'DOT' | 'IBTC' | 'INTR' | 'KBTC' | 'KINT' | 'KSM';
+import { CurrencyIdLiteral } from '@interlay/interbtc-api';
+
+type Tokens = 'BTC' | `${CurrencyIdLiteral}` | 'LKSM';
```

Comment:
I found that `'DOT' | 'IBTC' | 'INTR' | 'KBTC' | 'KINT' | 'KSM'` was duplicated.

Reply:
Hi @crypto-engineer - not keen on this change. I don't mind the duplication, because it makes things a bit clearer. There should also be no dependencies between interbtc-api and the component library.

Reply:
Hi @crypto-engineer - not keen on this change. I don't mind the duplication, because it makes things a bit clearer. There should also be no dependencies between interbtc-api and the component library.